docs/framework/configure-apps/file-schema/clear-element-for-configsections.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "&lt;clear&gt; element for &lt;configSections&gt; | Microsoft Docs"
+ms.date: "05/01/2017"
+ms.prod: ".net-framework"
+ms.technology: 
+  - "dotnet-clr"
+ms.topic: "article"
+f1_keywords: 
+  - "http://schemas.microsoft.com/.NetConfiguration/v2.0#configuration/configSections/clear"
+dev_langs: 
+  - "VB"
+  - "CSharp"
+  - "C++"
+  - "jsharp"
+helpviewer_keywords: 
+  - "clear Element"
+  - "<clear> Element"
+ms.assetid: 77f1d761-ff45-4001-8f36-3a3e5c41fa63
+author: "guardrex"
+ms.author: "mairaw"
+manager: "wpickett"
+---
+
+# \<clear> element for \<configSections>
+
+Clears all previously defined sections and section groups.
+
+[**\<configuration>**](~/docs/framework/configure-apps/file-schema/configuration-element.md)   
+&nbsp;&nbsp;[**\<configSections>**](~/docs/framework/configure-apps/file-schema/configsections-element-for-configuration.md)   
+&nbsp;&nbsp;&nbsp;&nbsp;**\<clear>**
+
+## Syntax
+
+```xml
+<clear/>
+```
+
+## Remarks
+
+The **\<clear>** element removes all sections and section groups from your application that were defined earlier in the current configuration file or at a higher level in the configuration file hierarchy.
+
+## Example
+
+The following example shows how to use the **\<clear>** element in an application configuration file to clear sections previously defined in the machine configuration file.
```

Comment:
Maybe make the forward reference here a little bit clearer: This example defines a machine configuration file and an application configuration file and shows...